cmd/oras/internal/option/platform.go

```diff
@@ -31,7 +31,7 @@ type Platform struct {
 
 // ApplyFlags applies flags to a command flag set.
 func (opts *Platform) ApplyFlags(fs *pflag.FlagSet) {
-	fs.StringVarP(&opts.Platform, "platform", "", "", "fetch the manifest of a specific platform if target is multi-platform capable")
+	fs.StringVarP(&opts.Platform, "platform", "", "", "only fetch a specific platform if target is multi-platform capable, in the form of os[/arch][/variant][:os_version])")
```

Comment:
If target is a manifest list, the specified platform is selected.
If target is a manifest, the platform is ensured to match the specified platform.

```suggestion
	fs.StringVarP(&opts.Platform, "platform", "", "", "ensure fetching a manifest of a platform in the form of os[/arch][/variant][:os_version])")
```